src/library/Hooks/useValidatorFilters/index.tsx

```diff
@@ -209,7 +209,7 @@ export const useValidatorFilters = () => {
     if (
       !searchTerm ||
       !Object.values(validatorIdentities).length ||
-      !Object.values(validatorSupers).length
+      !validatorSupers
```

Comment:
The same as with the problem `{identitiesSynced && supersSynced && display !== null ? (`.

if `validatorSupers == {}` ➡️Identity doesn't work, because the length is checked